atArgumentParser.go

```diff
@@ -32,7 +32,11 @@ func (at *atArgumentParser) ParseData(data string) error {
 	code := []byte(splitString[0])
 	arguments := make([][]byte, len(splitString)-1)
 	for i := 1; i < len(splitString); i++ {
-		arguments[i-1], err = hex.DecodeString(splitString[i])
+		fragment := splitString[i]
```

Comment:
duplicated code - you could create a method for this, gets a string returns a corrected one.

Reply:
Done.